doc/changes.rst

```diff
@@ -1,6 +1,79 @@
 Changelog
 =========
 
+Release v0.2.1 (2020/XX/XX)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4025418.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4025418
+
+Highlights
+
+* 🎉 **Patch release with more tutorials and gallery examples!** 🎉
+* 📹 Add `Liam <https://github.com/liamtoney>`__'s `ROSES 2020 PyGMT talk <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ (#643)
+
+New Features
+
+* Wrap plot3d (#471)
+* Wrap grdfilter (#616)
+
+Enhancements
+
+* Let plot() accept record-by-record transparency (#626)
+* Refactor info to allow datetime inputs from xarray.Dataset and pandas.DataFrame tables (#619)
+
+Tutorials & Gallery
+
+* Add tutorial for pygmt.config (#482)
+* Add an example for different line styles (#604, #664)
+* Add a gallery example for varying transparent points (#654)
+* Add tutorial for pygmt.Figure.text (#480)
+* Add an example for scatter plots with auto legends (#607)
+* Improve colorbar gallery example (#596)
+
+Documentation Improvements
+
+* doc: Fix the description of grdcontour -G option (#681)
+* Refresh Code of Conduct from v1.4 to v2.0 (#673)
+* Add PyGMT Zenodo BibTeX entry to main README.md (#678)
+* Complete most of documentation for makecpt (#676)
+* Complete documentation for plot (#666)
+* Add "no_clip" to plot, text, contour and meca (#661)
+* Add common alias "verbose" (V) to all functions (#662)
+* Improve documentation of Figure.logo() (#651)
+* Add mini-galleries for methods and functions (#648)
+* Complete documentation of grdimage (#620)
+* Add common alias perspective (p) for plotting 3D illustrations (#627)
+* Add common aliases xshift (X) and yshift (Y) (#624)
+* Add common alias cores (x) for grdimage and other multi-threaded modules (#625)
+* Enable switching different versions of documentation (#621)
+* Add common alias transparency (-t) to all plotting functions (#614)
+
+Bug Fixes
+
+* Disallow passing arguments like -XNone to GMT (#639)
+
+Maintenance
+
+* Migrate PyPI release to GitHub Actions (#679)
+* Upload artifacts showing diff images on test failure (#675)
+* Add slash command "/format" to automatically format PRs (#646)
+* Add instructions to run specific tests (#660)
+* Add more tests for xarray grid shading (#650)
+* Refactor xfail tests to avoid storing baseline images (#603)
+* Add blackdoc to format Python codes in docstrings (#641)
+* Check and lint sphinx configuration file doc/conf.py (#630)
+* Improve Makefile to clean ``__pycache__`` directory recursively (#611)
+* Update release process and checklist template (#602)
+
+This release contains contributions from:
+
+* `Dongdong Tian <https://github.com/seisman>`__
+* `Wei Ji Leong <https://github.com/weiji14>`__
+* `Conor Bacon <https://github.com/hemmelig>`__
+* `carocamargo <https://github.com/carocamargo>`__
```

Comment:
@carocamargo Thanks again for your contribution to the PyGMT project. We're going to release PyGMT v0.2.1 soon. Please let us know if you want to use the github username or real name here.

Reply:
Btw, If you want your name to be on the author list at Zenodo (https://zenodo.org/record/4025418), please also open up a Pull Request to add your name to https://github.com/GenericMappingTools/pygmt/edit/master/AUTHORS.md (and provide us with your ORCID and affiliation). Also cc @hemmelig on this.